packages/delivery-react-native/test/make-safe.test.ts

```diff
@@ -0,0 +1,86 @@
+import makeSafe from '../make-safe'
+
+describe('delivery: react native makeSafe', () => {
+  it('leaves simple types intact', () => {
```

Comment:
Can we add more types to this test? e.g. `null` & `undefined` (as they are special cased), `Symbol`, maybe a `Map` or other non-array iterable?